Core/include/Acts/EventData/VectorMultiTrajectory.hpp

```diff
@@ -438,6 +435,10 @@ class VectorMultiTrajectory final
     m_measCov.resize(m_measCov.size() + measdim * measdim);
   }
 
+  void setUncalibratedSourceLink_impl(IndexType istate, SourceLink sourceLink) {
```

Comment:
Actually now, when I look at it, shouldn't it be `const SourceLink&`?

Reply:
I move it into the other function call. I suppose we could keep it a const ref and then we copy once in the backend, but I thought it might be a good idea to support moving all the way. 

What do you prefer?

Reply:
Ah, it ha been years, but I still forget about moving. All good as is.